ui-components/src/forms/Field.tsx

```diff
@@ -120,7 +120,11 @@ const Field = (props: FieldProps) => {
       </div>
       {props.subNote && <p className="field-sub-note">{props.subNote}</p>}
       {props.errorMessage && (
-        <ErrorMessage id={`${idOrName}-error`} error={props.error}>
+        <ErrorMessage
+          id={`${idOrName}-error`}
+          error={props.error}
+          className="mt-2 inline-block leading-5"
```

Comment:
could we potentially just add these class names to the `error-message` class if we want them everywhere? (+ keep the new prop)

Reply:
for sure! I like the thought. One reason I didn't do that is because that'd change every usage of the <ErrorMessage /> component - including the [HouseholdSizeField error](https://dev-storybook-bloom.netlify.app/?path=/story/forms-householdsizefield--error-household-size-field) and FormSigninErrorBox - and I'm not sure if we want that top margin on those components. I was feeling like I was touching more than I should've so I settled for this change. I'd rather make this change everything though. If you still think I should it, you better believe I'm gonna! lmk

Reply:
to me the style request is a nice improvement on what we have so i would love to make it on all the fields if testing them individually looks good!